config/rubocop.rails.yml

```diff
@@ -1,6 +1,7 @@
 AllCops:
   Exclude:
   - 'db/schema.rb'
+  - 'vendor/ruby/**/*'
```

Comment:
Thoughts on adding `node_modules`, `bin/yarn`, or other common JS related directories to Exclude? Or keep this as something that should be added in the local `.rubocop.yml`?